### PR TITLE
Prepare for npmjs release

### PR DESCRIPTION
**Pull request by rvl**
### Issue

ADP-787

### Description

Cleanups of the typescript stuff in preparation for a beta release.

### CODE REVIEW

File: jsapi/README.md [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
    `cardano-addresses-jsbits` package.
- * A Cabal package `cardano-addresses-jsapi` containing GHCJS
+
+* A Cabal package `cardano-addresses-jsapi` containing GHCJS
    foreign exports for translating Javascript function calls and
    values into Haskell function calls and values, and vice-versa.
- * An NPM package `cardano-addresses` containing NodeJS modules written in TypeScript, which thinly wraps the GHCJS foreign exports to make a proper API.
 
-## Quick Start
+* An NPM package `cardano-addresses` containing NodeJS modules
```
Comment (rhyslbw) [inline jsapi/README.md:19]:
```suggestion
* An NPM package `cardano-addresses` containing ES modules
```

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looks good, just a small documentation change

File: jsapi/README.md (hunk 1 above)
Comment (rvl) [reply jsapi/README.md]: OK fixed. Actually there are both CommonJS and ES modules.

**Review by hamishmack — COMMENTED** [review]
Looks good, except for the problem removing the reachable return.

File: nix/cardano-addresses-js.nix [hunk 2]
```diff
@@ -3,11 +3,45 @@
...
+      -e '/function h\$ghcjszmprimZCGHCJSziPrimziJSVal_con_e/d' \
+
+    sed_inplace out.js \
+      -e '/function h\$integerzmwiredzminZCGHCziIntegerziTypezi\(Jn\|Jp\|S\)zh_con_e/,+3d'
+
+    # Remove most of the double-return statements. Without this,
+    # the sheer number of "unreachable code after return statement"
+    # warnings slows down Firefox considerably!
+    for js in rts.js out.js lib.js; do
+      sed_inplace "$js" -e '/^[[:space:]]*return.*;/!b;n;/^[[:space:]]*return.*;/d'
```
[the remark this answers is not in the document]
Comment (rvl) [reply nix/cardano-addresses-js.nix:42]: Ah yes, thanks!

[the remark this answers is not in the document]
Comment (rvl) [reply nix/cardano-addresses-js.nix]: Fixed

**Pull request merged**